hera_sim/config/HERA_H2C_CONFIG.yaml

```diff
@@ -0,0 +1,67 @@
+foregrounds:
```

Comment:
I like this idea.

I think that this would involve substantial modifications to the `Defaults` class, but this approach agrees more with the concept behind the functionality implemented by the class. I can look into implementing this if we think it's worth pursuing.

Reply:
let's leave it for a while, and keep it in mind, since it will take a bit of work. I think most of the other ideas don't really depend on this, so it should be okay to leave it for a while to stew.